go.mod

```diff
@@ -5,6 +5,6 @@ go 1.16
 require (
 	github.com/google/uuid v1.2.0
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.0.1
-	github.com/tozny/e3db-clients-go v0.0.194
+	github.com/tozny/e3db-clients-go v0.0.198-0.20211129172200-06b838d3685d
```

Comment:
updated to current feature branch of e3db-clients-go. this should be fine so long as we don't release a new version of our provider before the client code gets checked in.